lib/eventboss/unit_of_work.rb

```diff
@@ -21,7 +21,17 @@ def run(client)
     rescue StandardError => exception
       handle_exception(exception, processor: processor, message_id: @message.message_id)
     else
-      cleanup(client)
+      cleanup(client) unless postponed_by
+    ensure
+      change_message_visibility(client) if @postponed_by
```

Comment:
This will run, even if there were errors. Is that intentional?

Reply:
Yes, since we want to change visibility on error. That's intentional.

Reply:
👍